liblttng-ust-java-agent/java/lttng-ust-agent-jul/org/lttng/ust/agent/jul/LttngLogHandler.java

```diff
@@ -134,13 +157,27 @@ public void publish(LogRecord record) {
 		 */
 		LttngJulApi.tracepointWithContext(formattedMessage,
 				record.getLoggerName(),
-				record.getSourceClassName(),
-				record.getSourceMethodName(),
+				getSourceClassName(record),
+				getSourceMethodName(record),
```

Comment:
Since the check is very simple you could use one-liners:

    ...
    record.getLoggerName(),
    logSourceClass ? record.getSourceClassName() : EMPTY,
    logSourceMethod ? record.getSourceMethodName() : EMPTY,
    record.getMillis(),
    ...